patches/extra/ungoogled-chromium/enable-paste-and-go-new-tab-button.patch

```diff
@@ -0,0 +1,34 @@
+--- a/chrome/browser/ui/views/tabs/new_tab_button.cc
++++ b/chrome/browser/ui/views/tabs/new_tab_button.cc
+@@ -63,10 +63,8 @@ const gfx::Size NewTabButton::kButtonSize{28, 28};
+ NewTabButton::NewTabButton(TabStrip* tab_strip, views::ButtonListener* listener)
+     : views::ImageButton(listener), tab_strip_(tab_strip) {
+   set_animate_on_state_change(true);
+-#if defined(OS_LINUX) && !defined(OS_CHROMEOS)
+   set_triggerable_event_flags(triggerable_event_flags() |
+                               ui::EF_MIDDLE_MOUSE_BUTTON);
```

Comment:
Just to be certain, is this constant defined in macOS and Windows?

Reply:
https://github.com/chromium/chromium/blob/2d57e5b8afc6d01b344a8d95d3470d46b35845c5/chrome/browser/ui/views/toolbar/toolbar_view.cc#L174-L222

This is the place where the `ui::EF_MIDDLE_MOUSE_BUTTON` is actively used for all platforms.

Reply:
Alright, thanks for pointing that out